plugins/fedora-pkgdb-collections/gs-plugin-fedora-pkgdb-collections.c

```diff
@@ -221,39 +221,10 @@ gs_plugin_refresh (GsPlugin *plugin,
 static gchar *
 _get_upgrade_css_background (guint version)
 {
-	g_autoptr(GSettings) settings = NULL;
 	g_autofree gchar *version_str = g_strdup_printf ("%u", version);
 	g_autofree gchar *filename0 = NULL;
 	g_autofree gchar *filename1 = NULL;
 	g_autofree gchar *filename2 = NULL;
-	g_autofree gchar *uri = NULL;
-
-	settings = g_settings_new ("org.gnome.software");
-	uri = g_settings_get_string (settings, "upgrade-background-uri");
-	if (uri != NULL && *uri != '\0') {
-		const gchar *ptr;
-		guint percents_u = 0;
-		for (ptr = uri; *ptr != '\0'; ptr++) {
-			if (*ptr == '%') {
-				if (ptr[1] == 'u')
-					percents_u++;
-				else if (ptr[1] == '%')
-					ptr++;
-				else
-					break;
-			}
-		}
-
-		if (*ptr != '\0' || percents_u > 3) {
-			g_warning ("Incorrect upgrade-background-uri (%s), it can contain only up to three '%%u' sequences", uri);
-		} else {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
-			filename1 = g_strdup_printf (uri, version, version, version);
-#pragma GCC diagnostic pop
-			return g_strdup_printf ("url('%s')", filename1);
-		}
```

Comment:
Fascinating. What's wrong with using `%1$u` in each place where the URL is needed? Ah well.